lib/tasks/avalon_migrations.rake

```diff
@@ -114,5 +114,18 @@ namespace :avalon do
         puts "Migration completed successfully."
       end
     end
+    desc "Backfill original filename property on MasterFile records"
+    task master_file_original_filename: :environment do
+      count = 0
+      missing = MasterFile.where(original_filename: nil)
+      missing.each do |mf|
```

Comment:
I'm fine with merging this now, but as part of testing we should test that this works and is performant when running on a large repository.